apps/nextjs-website/src/components/atoms/BannerLink/BannerLink.tsx

```diff
@@ -0,0 +1,129 @@
+import { Box, Stack, Typography, useTheme } from '@mui/material';
+import { BlocksContent, BlocksRenderer } from '@strapi/blocks-react-renderer';
+import Image from 'next/image';
+import IconWrapper from '../IconWrapper/IconWrapper';
+import { Media } from '../../../lib/strapi/codecs/MediaCodec';
+import { FC } from 'react';
+
+export type BannerLinkProps = {
+  contentMaxWidth?: number;
+  justify?: string;
+  content?: BlocksContent;
+  title: string;
+  icon: Media;
+  theme: 'light' | 'dark';
+  count?: number;
+  boxMediumPadding?: string;
+  centerBannerWidth?: number;
+};
+
+export const BannerLink: FC<BannerLinkProps> = ({
+  contentMaxWidth = 450,
+  theme,
+  content,
+  icon,
+  title,
+  justify = 'center',
+  count = 2,
+  boxMediumPadding = '0px 130px',
+  centerBannerWidth = count <= 2 ? 700 : 466,
+}) => {
+  const { palette } = useTheme();
+
+  if (!content) return null;
+  const backgroundColor =
+    theme === 'dark' ? palette.primary.dark : palette.primary.light;
+  const textColor = palette.primary.contrastText;
+  const width = justify === 'center' && count > 1 ? centerBannerWidth : '100%';
+  return (
+    <Box
+      bgcolor={backgroundColor}
+      component='section'
+      maxWidth={{ xs: '100%', md: width }}
+      sx={{ width: '100%', direction: 'column' }}
+      justifyContent={{ xs: 'center', md: justify }}
+      style={{
+        display: 'flex',
+        textAlign: 'center',
+      }}
```

Comment:
You are managing box layout with four different props, I think you should use only a single property here
```suggestion
      sx={{
        direction: 'column',
        display: 'flex',
        justifyContent: { xs: 'center', md: justify },
        maxWidth: { xs: '100%', md: width },
        textAlign: 'center',
        width: '100%',
      }}
```